README.md

```diff
@@ -30,145 +30,144 @@ Styleguide for low-fidelity paper prototyping and wireframing.
 
 ### 4. Blue elements 💙 Features
 
-4.1. _MUST_. Use the **blue** color to describe the content, origin and destination of features.
+4.1. _MUST_ use **blue** color to describe features' content, origin and destination.
 
-4.2. _MUST_. Define a **reference code** for each feature with a letter, starting from `A` to `Z`.
+4.2. _MUST_ define **reference codes** for each feature with a letter, starting from `A` to `Z`.
 
 ![](https://raw.githubusercontent.com/NOALVO/wireguide/master/img/42.PNG)
 
-4.3. _SHOULD NOT_. Prefer omiting control labels and instead use the reference code described in 4.2.
+4.3. _SHOULD NOT_ label control elements directly. Instead, prefer using the reference code described in 4.2.
 
-4.4. _MUST_. Build a "**Table of Features**" including the reference code and the main details of each in a usable free space of the wireframe. If there is no free space in the wireframe, use the back of the sheet to make it.
+4.4. _MUST_ make a "**features table**" including each component reference code and their main details. Use any free space in the wireframe. If there isn't, use the back of the sheet to make it.
 
 ![](https://raw.githubusercontent.com/NOALVO/wireguide/master/img/44.PNG)
 
-4.5. _MUST_. When referencing features in Table of Features specified by item 4.4, **use `"` to describe their labels**.
+4.5. _MUST_ **use double quotes (`"`) to describe labels** of items in a features table (specified by item 4.4).
 
 ![](https://raw.githubusercontent.com/NOALVO/wireguide/master/img/45.PNG)
 
-4.6. _MUST_. When referencing features in Table of Features specified by item 4.4, **use `→` to indicate the destination of the feature**, like, but not limited to, opening dialogs and changing pages.
+4.6. _MUST_ **use right arrows (`→`) to indicate a feature's destination** when referencing it in a features table (specified by item 4.4). Like, but not limited to, opening dialogs and changing pages.
 
 ![](https://raw.githubusercontent.com/NOALVO/wireguide/master/img/46.PNG)
 
-4.7. _MUST_. When referencing features in Table of Features specified by item 4.4, **use `←` to indicate the origin of the feature**, like, but not limited to, pages e other features, and also the source of labels, like, but not limited to APIs and internal funcions.
+4.7. _MUST_ **use left arrows (`←`) to indicate a feature's origin** when referencing it in a features table (specified by item 4.4). Like, but not limited to, pages and other features, and also the labels' sources, like, but not limited to APIs and its internal functions.
 
 ![](https://raw.githubusercontent.com/NOALVO/wireguide/master/img/47.PNG)
 
 ### 5. Red elements ❤️ Modifiers
 
-5.1. _MUST_. Use the **red** color to indicate special behaviors of features.
+5.1. _MUST_ use **red** color to indicate features' special behaviors.
 
-5.2. _MUST_. Mark read-only, blocked and disabled controls with an "block" sign.
+5.2. _MUST_ use "blocked" signs (`🚫`) for read-only, blocked or disabled controls.
```

Comment:
"Use" is not the proper word for these modifiers. Modifiers are not controls or containers. Controls and containers can be USED, but only can be MARKED with a modifier. If it is not the right word, please suggest a synonym of "mark".

Reply:
Put `marked` back